X/XNNPACK/build_tarballs.jl

```diff
@@ -74,17 +71,22 @@ install_license ../LICENSE
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = supported_platforms()
-filter!(p -> !(Sys.iswindows(p) && arch(p) == "i686"), platforms) # i686-Windows fails to link
+filter!(p -> arch(p) != "armv6l", platforms) # armv6l is unsupported by XNNPACK (lacks NEON instructions)
 filter!(p -> arch(p) != "powerpc64le", platforms) # PowerPC64LE is unsupported by XNNPACK (Unsupported architecture in src/init.c)
+platforms = expand_microarchitectures(platforms, [
```

Comment:
Please don't use it, it doesn't work.

Reply:
OK :-)

Reply:
Ah - it actually enables the armv7l build (for neonvfpv4). Is that acceptable? (just one micro-architecture, for a non-tier-1 platform)

Reply:
The whole microarchitecture stuff doesn't work. Specifically, the artifact selection in Pkg is broken. There is a reason why there isn't a single builder using it

Reply:
OK - removing it.